spec/controllers/proposals_controller_spec.rb

```diff
@@ -229,9 +229,8 @@
       token = create(:api_token, step: approval)
       approval.user.add_delegate(create(:user))
 
-      post :approve, id: proposal.id, cch: token.access_token
+      get :approve, id: proposal.id, cch: token.access_token
```

Comment:
I changed the `post` to a `get` in order to more closely represent the actual use case, which is a delegate clicking on a link in an email.

The HTTP method used in each spec should be specific to the particular use case being tested. I'm not sure what kind of grouping you have in mind? Maybe we could defer that as outside the scope of this change?